src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditAccommodationCommand.java

```diff
@@ -30,15 +31,16 @@
             + ": Edits the accommodation identified by the index number used in the displayed accommodation list.\n"
             + "Parameters: INDEX (must be a positive integer)\n"
             + "[" + PREFIX_NAME + "NAME] "
+            + "[" + PREFIX_LOCATION + "LOCATION] "
+            + "[" + PREFIX_COST + "COST]"
             + "[" + PREFIX_START + "STARTDATE] "
             + "[" + PREFIX_END + "ENDDATE] "
-            + "[" + PREFIX_LOCATION + "LOCATION] "
             + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_NAME + "Hard Rock Hotel"
-            + PREFIX_START + " jan 20 2020 "
-            + PREFIX_END + " jan 30 2020"
-            + PREFIX_LOCATION + "Sentosa"
-            + PREFIX_COST + "SGD500";
+            + PREFIX_NAME + "Hard Rock Hotel "
+            + PREFIX_LOCATION + "Sentosa "
+            + PREFIX_COST + "$500 "
```

Comment:
I think the cost is just numbers? iirc

Reply:
yes it is supposed to be numbers, i will change it